packages/@react-spectrum/combobox/src/ComboBox.tsx

```diff
@@ -179,7 +179,6 @@ function ComboBox<T extends object>(props: SpectrumComboBoxProps<T>, ref: RefObj
         }
         style={{width: '100%'}}>
         <TextFieldBase
-          {...otherProps}
```

Comment:
removed cuz styleProps that should only be applied to the combobox div (e.g. margins and stuff) also got passed to textfield